### PR TITLE
Hbase monitoring metadata browse API in new app framework

### PR DESCRIPTION
**Pull request by yonzhang**
https://issues.apache.org/jira/browse/EAGLE-428

### CODE REVIEW

**Comment by haoch** [comment]
LGTM, except:

1. MetadataDaoFactory, but we could refactor this later
2. Prefer to rename configuration key "connection" as "metadata.connection"

If we want to use json-style configuration, we should well org configuration by `{}` object group.

File: eagle-server/src/main/resources/application.conf [hunk 1]
```diff
@@ -42,6 +42,7 @@
 	"metadata":{
 		"store": "org.apache.eagle.metadata.service.memory.MemoryMetadataStore"
 	},
+	"connection":"jdbc:mysql://server.eagle.apache.org:3306/eagle",
```
Comment (haoch) [inline eagle-server/src/main/resources/application.conf]: Could we move the `connection` into `metadata.connection`?

Comment (RalphSu) [inline eagle-server/src/main/resources/application.conf]: Suggest use nest {}

Comment (haoch) [inline eagle-server/src/main/resources/application.conf]: Yes, I meant

```
metadata{
   connection: "jdbc:mysql://server.eagle.apache.org:3306/eagle"
}
```

**Comment by yonzhang** [comment]
That is obvious. Again this is to make first use case to work, I will leave this change to app framework refine.

**Pull request closed without merging**